packages/chargebee/src/constants/index.jsx

```diff
@@ -0,0 +1,3 @@
+export const SECTION_INVOICES = 'BILLING_HISTORY';
```

Comment:
probably best to store all possible sections here and also leave comment with a link to the docs page

Reply:
added link to doc. Let's add constants as we need them